packages/metastream-app/src/locale/tr.ts

```diff
@@ -0,0 +1,134 @@
+export default {
+  add: 'Ekle',
+  addMedia: 'Link Ekle',
+  admin: 'Yönetici',
+  advanced: 'Gelişmiş',
+  allow: 'İzin Ver',
+  allowTracking: 'Uygulama kullanım bilgilerini gönder.',
+  allowTrackingDesc:
+    'Anonim kullanım bilgileriniz uygulama kullanımının daha iyi anlaşılabilmesi için geliştiriciye gönderilecektir.',
+  always: 'Her zaman',
+  appearance: 'Görünüm',
+  autoFullscreen: 'Otomatik tam ekran',
+  autoFullscreenDesc: 'Oynatılan videoyu pencere alanına sığdırmak için yakınlaştırır.',		   
+  avatar: 'Avatar',
+  avatarCredit: '<1>{{artist}}</1> tarafından tasarlanan avatar seçildi.',
+  cancel: 'İptal et',
+  changelog: 'Son değişiklikler',
+  chat: 'Sohbet',
+  chatColor: 'Sohbet rengi',
+  chatDockToRight: 'Ekranın sağına sabitle',
+  chatMessageFailed: 'Mesajınız gönderilemedi, lütfen tekrar deneyiniz.',
+  chatPlaceholder: 'Mesaj ya da bağlantı ekle',
+  chatRevealHint: 'Sohbete erişmek için <1></1>'a basın.',
+  chatNewMessage: 'Yeni bir mesaj gönderildi!',
+  chatUndock: 'Sohbeti eski yerine al',
+  chatTyping1: '<0></0> yazıyor',
+  chatTyping2: '<0></0> ve <2></2> yazıyor',
+  chatTyping3: '<0></0>, <2></2>, ve <4></4> yazıyor…',
+  chatTypingSeveral: 'Birçok kullanıcı yazıyor…',
+  chooseDisplayName: 'Sohbette görünecek adınızı seçiniz.',
+  connecting: 'Bağlanıyor',
+  continue: 'Devam et',
+  copy: 'Kopyala',
+  copyLink: 'Linki kopyala',
+  debug: 'Debug',
+  developerMode: 'Geliştirici Modu',
+  developerModeDesc: 'Web sayfalarını görüntüleyebilmek için geliştirici araçlarını göster.',
+  directIP: 'Direkt IP',
+  disallow: 'İzin verme',
+  disconnect: 'Bağlantıyı kes',
+  disconnected: 'Bağlantı kesildi',
+  displayName: 'Görünen ad',
+  dj: 'DJ',
+  donate: 'Bağış yap',
+  donators: 'Bağışçılar',
+  duplicate: 'Kopyala',
+  endSessionTitle: 'Oturumu kapatmak istediğine emin misiniz?',
+  endSessionDescription: 'Özel oturuma geçmek bütün kullanıcıların bağlantılarının kesilmesine yol açacaktır.',
+  endSessionModalDescription: 'Bütün kullanıcıların oturum ile bağlantıları kesilecektir.',
+  endSessionButton: 'Oturumu kapat',
+  enterJoinDest: 'Bir Metastream oturum URLsi giriniz.',
+  findSession: 'Oturumu bul',
+  findSessionDescription:
+    'Metastream Discord'undaki <1>#sessions</1> kanalına katılarak diğer kullanıcıların paylaştığı oturumlara erişebilirsiniz. Katılmak için Discord logosuna tıklayınız.',
```

Comment:
Extra `'` here. If this is a necessary for grammar, it can be replaced with `\'` to escape the quote as per [JavaScript's escape notation.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#Escape_notation)

Reply:
Oh ok sorry I'm not really familiar about the coding aspects of this kind of things.